client/prototypes.h

```diff
@@ -21,6 +21,16 @@
 #ifndef __CLIENT_PROTOTYPES_H__
 #define __CLIENT_PROTOTYPES_H__
 
+#include <openssl/sha.h>
+#include <openssl/md5.h>
+
+typedef struct metalinkfile{
+    char *filename;
+    int   type;
```

Comment:
change the type from int to enum TDNF_HashType_e.

Reply:
Not addressing this right now.